cli/command/service/logs.go

```diff
@@ -112,7 +112,7 @@ func runLogs(dockerCli *command.DockerCli, opts *logsOptions) error {
 			return err
 		}
 		maxLength = getMaxLength(task.Slot)
-		responseBody, err = cli.TaskLogs(ctx, opts.target, options)
+		responseBody, _ = cli.TaskLogs(ctx, opts.target, options)
```

Comment:
Finally, delete this second call to `cli.TaskLogs` entirely. I have no idea how this got here. We shouldn't be calling this twice.